package.json

```diff
@@ -23,14 +23,14 @@
     "url": "https://github.com/apostrophecms/eslint-config-apostrophe/issues"
   },
   "homepage": "https://github.com/apostrophecms/eslint-config-apostrophe#readme",
-  "peerDependencies": {
-    "eslint": ">= 8.39.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-n": "^16.0.0",
-    "eslint-plugin-promise": "^6.1.1"
-  },
   "dependencies": {
-    "eslint-config-standard": "^17.1.0"
+    "eslint": "^8.53.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.29.0",
+    "eslint-plugin-n": "^16.2.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-standard": "^4.0.1"
```

Comment:
Tried to add all the dependencies here to not have to install them in all are projects, not sure it works though, should we put it in the peedDeps instead?